plugins/inputs/smart/smart.go

```diff
@@ -36,18 +36,8 @@ var (
 	// PASSED, FAILED, UNKNOWN
 	smartOverallHealth = regexp.MustCompile("^(SMART overall-health self-assessment test result|SMART Health Status):\\s+(\\w+).*$")
 
-	// Accumulated start-stop cycles:  7
-	sasStartStopAttr = regexp.MustCompile("^Accumulated start-stop cycles:\\s+(.*)$")
-	// Accumulated load-unload cycles:  39
-	sasLoadCycleAttr = regexp.MustCompile("^Accumulated load-unload cycles:\\s+(.*)$")
-	// Current Drive Temperature:     34 C
-	sasTempAttr = regexp.MustCompile("^Current Drive Temperature:\\s+(.*)\\s+C(.*)$")
-	// Temperature: 38 Celsius
-	nvmeTempAttr = regexp.MustCompile("^Temperature:\\s+(.*)\\s+(.*)$")
-	// Power Cycles: 472
-	nvmePowerCycleAttr = regexp.MustCompile("^Power Cycles:\\s+(.*)$")
-	// Power On Hours: 6,038
-	nvmePowerOnAttr = regexp.MustCompile("^Power On Hours:\\s+(.*)$")
+	// sasNvmeAttr is a SAS or NVME SMART attribute
+	sasNvmeAttr = regexp.MustCompile(`^([^:]*):\s+(.*)$`)
```

Comment:
May be useful to require submatch strings have a length > 0: `^([^:]+):\s+(.+)$`